src/Chainweb/SPV/CreateProof.hs

```diff
@@ -259,6 +260,27 @@ outputProofPrefix i db payload = do
   where
     blockOutputTable = _payloadCacheBlockOutputs $ _payloadCache db
 
+-- TODO: incorporate version into crossnet: syntax?
+-- TODO: if and when this is less trivial, incorporate this logic into createPayloadProof
+createCrossNetworkTransactionOutputProof :: (CanReadablePayloadCas tbl, HasCallStack) => CutDb tbl -> Text -> ChainId -> BlockHeight -> Int -> IO (TransactionOutputProof SHA512t_256)
+createCrossNetworkTransactionOutputProof cutDb tgt scid txHeight txIx = do
+    tip <- maxEntry headerDb
+    txHeader <- maybe (throwM headerNotFound) pure =<< seekAncestor headerDb tip (int txHeight)
+    Just payload <- tableLookup pDb (_blockPayloadHash txHeader)
```

Comment:
Can we get better error reporting if the lookup fails here?

Reply:
I will add it, I did this because there's identical code elsewhere and a missing payload is a "we're totally screwed, database is corrupt" situation. But I wouldn't mind a better error anyway.